SharedUtils/src/main/java/com/braintreepayments/api/HttpResponseCallback.kt

```diff
@@ -7,7 +7,7 @@ import androidx.annotation.RestrictTo
  * @suppress
  */
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-interface HttpResponseCallback {
+fun interface HttpResponseCallback {
```

Comment:
Nice 😎 - tech debt item but we should really do this with all of the SDK interfaces